src/custom/Modal/index.tsx

```diff
@@ -57,7 +57,7 @@ const StyledDialog = styled(Dialog)`
 const StyledHeader = styled('div')(({ theme }) => ({
   background:
     theme.palette.mode === 'light'
-      ? 'linear-gradient(90deg, #3B687B 0%, #507D90 100%)'
```

Comment:
please ensure you follow the latest modals style for header and footer : [See here](https://www.figma.com/design/ekm0Hh6M25bSKpP5BcIJom/Sistent-Design-System?node-id=5209-13685&t=HV6VZ8j9RqnrcALS-0)